src/libs/brands.ts

```diff
@@ -290,6 +328,61 @@ export const brands: Record<string, BrandConfiguration> = {
       colorPalette: { ...baseColors, primary: '#521b93', secondary: '#011c48', accent: '#521b93' },
     },
   },
+  publicHealth: {
+    name: 'Terra for Public Health Genomics',
+    queryName: 'publicHealth',
+    welcomeHeader: 'Terra for Public Health Genomics',
+    description:
+      'Terra is a secure cloud-based data platform enabling pathogen genomic data analysis in public health, clinical, and academic settings.',
+    hostName: 'publichealth.terra.bio',
+    docLinks: [
+      {
+        link: 'https://dockstore.org/organizations/Theiagen',
+        text: 'Theiagen Workflow Repository',
+      },
+      {
+        link: 'https://docs.google.com/document/d/e/2PACX-1vTPQC-bYMRxXQ4Gz9ESH_Eo-E6UXD2qOL7_3iMxJaQF3pOyW3tUyu7G9Nvk-JTf8xDkOyhftvd9L-sa/pub',
+        text: 'Terra Cloud Costs FAQs',
+      },
+      {
+        link: 'mailto:publichealthgenomics@broadinstitute.org',
+        text: 'Contact us',
```

Comment:
Super nit: this line is not like the others in that it's a verb while the others are nouns and it's the only sentence case link.

Also, would moving this to the bottom of the list suggest "try Terra Support before emailing us"?

Reply:
Hm I'm hesitant to change this since the verbiage/order was decided on by UX and the folks requesting the new branded site